src/library/components/recent-activity/index.js

```diff
@@ -34,6 +34,7 @@ const offeringMapping = data => {
     completedStudentsCount: completedStudents.length,
     reportUrl: data.report_url,
     externalReport: externalReportMapping(data.external_report),
```

Comment:
I think this line with the old singular `externalReport` declaration should be removed right?